docs/knowledgebase/advanced/account-info.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Account Information
+---
+
+## `Account` StorageMap
+
+In Substrate, account information is stored in a
+[**Storage Map**](https://substrate.dev/rustdocs/v3.0.0/frame_support/storage/types/struct.StorageMap.html#impl),
+extracted and shown below:
+
+[`frame/system/src/lib.rs`](https://github.com/paritytech/substrate/blob/bcd649ffca9efc93f8b4ac1506ec8117b71e1aac/frame/system/src/lib.rs#L530-L538):
+
+```rust
+/// The full account information for a particular account ID.
+#[pallet::storage]
+#[pallet::getter(fn account)]
+pub type Account<T: Config> = StorageMap<
+  _,
+  Blake2_128Concat,
+  T::AccountId,
+  AccountInfo<T::Index, T::AccountData>,
+  ValueQuery,
+>;
+```
+
+`Account` storage map takes in five type parameters, so it has no prefix, use `Blake2_128Concat` as
```

Comment:
wtf does this mean?

Reply:
I think this is a misuse of "so" as a connective; here I think it means "of which" instead of "as a result".

IIRC this is also factually incorrect, having `_` as its prefix doesn't mean that it has no prefix. [During macro parsing](https://github.com/paritytech/substrate/blob/90cfb952f2e11bc6d327faf04b30dc5f4cf89df8/frame/support/procedural/src/pallet/parse/storage.rs#L383-L390), we actually expect this to be `_` at all times, which then [gets replaced by the `Ident` of the storage as the storage prefix](https://github.com/paritytech/substrate/blob/90cfb952f2e11bc6d327faf04b30dc5f4cf89df8/frame/support/procedural/src/pallet/parse/storage.rs#L82).

Reply:
Yes, this is correct, there is ALWAYS a prefix

Reply:
Thanks for clarification @KiChjang. Fixed.